uk.co.compucorp.civicrm.hrcore/tests/phpunit/drupal_function_mocks.php

```diff
@@ -35,6 +35,12 @@ function user_load($userID) {
   $user = new stdClass();
   $user->roles = [1 => 'Fake Role'];
   $user->uid = $userID;
+  $user->status = 1;
+
+  if($userID == 0) {
+    $user->status = 0;
+  }
+
```

Comment:
I'll approve this change, but this type of change is not very maintainable. Eventually, we'll need to come up with a better solution for mocking this functions. I think we can implement some sort of in-memory database that can be used by these user_ functions to keep track of users